x/perpetual/client/cli/query_pool_test.go

```diff
@@ -152,3 +152,4 @@ func TestListPool(t *testing.T) {
 		)
 	})
 }
+*/
```

Comment:
can we uncomment it ?

Reply:
Hello Amit. I commented this because this kind of test when the cli is involved and more of a keeper too. They failed If we can keep it like technical debt in order to can close the task please and attack that more later.

Reply:
got it, so if this test is failing, is query working ? have you tested it on localnet ?

Reply:
Yes @amityadav0  the query is working. I tested it on my localnet.  Fail is the init of the chain with a timeout not the query .

Reply:
![image](https://github.com/user-attachments/assets/4b668ec6-9f4c-4180-b976-d5fb8d139575)

Reply:
The not oracle pools are not listed:

`{"pool":[{"amm_pool_id":"2","health":"0.883351042402733204","enabled":true,"closed":false,"borrow_interest_rate":"0.000000535466554908","pool_assets_long":[{"liabilities":"3931813932907","custody":"0","take_profit_liabilities":"3605857570107","take_profit_custody":"0","asset_balance":"1062716517114","block_borrow_interest":"0","asset_denom":"ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65"},{"liabilities":"0","custody":"563524739564","take_profit_liabilities":"0","take_profit_custody":"394720462935","asset_balance":"-563584155567","block_borrow_interest":"0","asset_denom":"ibc/E2D2F6ADCC68AA3384B2F5DFACCA437923D137C14E86FB8A10207CF3BED0C8D4"}],"pool_assets_short":[{"liabilities":"0","custody":"25521549362","take_profit_liabilities":"0","take_profit_custody":"758800649","asset_balance":"-13515873900","block_borrow_interest":"0","asset_denom":"ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65"},{"liabilities":"3173650598","custody":"0","take_profit_liabilities":"758578518","take_profit_custody":"0","asset_balance":"0","block_borrow_interest":"0","asset_denom":"ibc/E2D2F6ADCC68AA3384B2F5DFACCA437923D137C14E86FB8A10207CF3BED0C8D4"}],"last_height_borrow_interest_rate_computed":"9864089","funding_rate":"0.001000000000000000"},{"amm_pool_id":"4","health":"1.000010318825890871","enabled":true,"closed":false,"borrow_interest_rate":"0.000000219967299502","pool_assets_long":[{"liabilities":"-200000000","custody":"0","take_profit_liabilities":"-186115695","take_profit_custody":"0","asset_balance":"6781429","block_borrow_interest":"0","asset_denom":"ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65"},{"liabilities":"0","custody":"0","take_profit_liabilities":"0","take_profit_custody":"-445632798","asset_balance":"0","block_borrow_interest":"0","asset_denom":"ibc/B4314D0E670CB43C88A5DCA09F76E5E812BD831CC2FEC6E434C9E5A9D1F57953"}],"pool_assets_short":[{"liabilities":"0","custody":"0","take_profit_liabilities":"0","take_profit_custody":"0","asset_balance":"320805","block_borrow_interest":"0","asset_denom":"ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65"},{"liabilities":"0","custody":"0","take_profit_liabilities":"-1","take_profit_custody":"0","asset_balance":"0","block_borrow_interest":"0","asset_denom":"ibc/B4314D0E670CB43C88A5DCA09F76E5E812BD831CC2FEC6E434C9E5A9D1F57953"}],"last_height_borrow_interest_rate_computed":"9864089","funding_rate":"0.000000000000000000"}],"pagination":{"next_key":null,"total":"0"}}`